cognitedata/reveal
# feat: add partitioning and pagination support for fetching events

#### Type of change

![Feat](https://img.shields.io/badge/Type-Feat-green)  

#### Jira ticket :blue_book:


https://cognitedata.atlassian.net/browse/REV-718

## Description :pencil:


Adds support for > 1000 events for a single collection. This also leverages partitioning, to parallel-fetch a lot of events.

## Test instructions :information_source:



On 3d-test, use the siteId: `celanese1`. Which contains more than 1000 events. And see that they are all loaded.
